packages/css-framework/src/components/_checkbox.scss

```diff
@@ -80,6 +80,7 @@ $btn-focus-width: 1px;
   width: 100%;
   height: 100%;
   line-height: 1;
+  font-size: 14px;
```

Comment:
Can the `f.font-size` helper be used here?

Reply:
I had to set an explicit size in `px` to prevent some weird alignment issues with the tick character.